### PR TITLE
Tests: Bump pruning size value for unit test

### PR DESCRIPTION
**Pull request by brbrr**
`Update_known_chain_sizes` is failing on master, which blocks PR merging to master.

## Changes

- Increase boundary prunning size value for the `Update_known_chain_sizes` test

#### What types of changes does your code introduce?

- (Temporary) Fix for a unit test 

#### Requires testing

- [ ] Yes
- [x] No

### CODE REVIEW

**Review by tkstanczak — CHANGES_REQUESTED** [review]
Add test. What would be a good test? Maybe it should fail if there was not update to this file for more than a month?

**Comment by brbrr** [comment]
From a QA standpoint, the test is fragile by design since it relies on data outside of the test controls. Thus, the behavior of the test cannot be deterministic. Having a test that checks the file updates might work if the pruning size is growing linearly, which I don't know. 

Ideally, we'd fetch the expected values on every test run, but it may defeat the purpose of the test :)

Anyway, currently tests on the master are failing, which means all the PRs are blocked. Maybe we can merge it as-is, and then create a follow-up issue/PR with a long-term solution?

**Review by Demuirgos — APPROVED** [review]

**Review by LukaszRozmej — APPROVED** [review]
All this are estimates on moving targets, not sure how would you test it

File: src/Nethermind/Nethermind.Blockchain.Test/KnownChainSizesTests.cs [hunk 1]
```diff
@@ -14,7 +14,7 @@ public class KnownChainSizesTests
     public void Update_known_chain_sizes()
     {
         // Pruning size have to be updated frequently
-        ChainSizes.CreateChainSizeInfo(BlockchainIds.Mainnet).PruningSize.Should().BeLessThan(200.GB());
+        ChainSizes.CreateChainSizeInfo(BlockchainIds.Mainnet).PruningSize.Should().BeLessThan(250.GB());
```
Comment (deffrian) [inline src/Nethermind/Nethermind.Blockchain.Test/KnownChainSizesTests.cs:17]: imo +25% is too big for hotfix

**Review by deffrian — CHANGES_REQUESTED** [review]

File: src/Nethermind/Nethermind.Blockchain.Test/KnownChainSizesTests.cs (hunk 1 above)
Comment (deffrian) [reply src/Nethermind/Nethermind.Blockchain.Test/KnownChainSizesTests.cs:17]: 10gb should be fine

Comment (brbrr) [reply src/Nethermind/Nethermind.Blockchain.Test/KnownChainSizesTests.cs]: resolved in [f379e33](https://github.com/NethermindEth/nethermind/pull/6087/commits/f379e33bedba522f3fef094b2bba0c3a238a9e15)

**Review by deffrian — APPROVED** [review]

**Review by rubo — APPROVED** [review]

**Comment by tkstanczak** [comment]
@LukaszRozmej you can write a test that checks with github hidden folder to verify when the file got last time updated. And fail if it was not updated recently enough.

**Comment by tkstanczak** [comment]
Oh, this is a test file already? Sorry, I thought ir was some production code.

**Pull request merged**